### PR TITLE
11.0 mig l10n nl dutch company type

### PR DESCRIPTION
**Pull request by lfreeke**

### CODE REVIEW

**Comment by hbrunn** [comment]
please fix https://travis-ci.org/OCA/l10n-netherlands/jobs/458460707#L568

this depends on `partner_company_type` which isn't yet migrated in v11

**Comment by lfreeke** [comment]
@hbrunn  https://github.com/OCA/partner-contact/pull/660 there is a pull request for partner_company_type. Should I wait until that is merged in?

**Comment by hbrunn** [comment]
yes, we can only go further here if all dependencies are merged

File: l10n_nl_dutch_company_type/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Copyright 2018 Therp BV <https://therp.nl>
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/__init__.py]: Do away with Copyright line from __init__.py files

File: l10n_nl_dutch_company_type/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "l10n_nl_dutch_company_type",
+    "version": "11.0.1.0.0",
+    "author": "Therp BV, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Dutch localization",
+    "summary": "Adds the Dutch company types",
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/__manifest__.py:9]: Add website key with url https://github.com/OCA/l10n-netherlands

File: l10n_nl_dutch_company_type/readme/CREDITS.rst [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[ This file is optional and contains additional credits, other than
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/CREDITS.rst]: Remove this file and then regenerate README.rst

File: l10n_nl_dutch_company_type/readme/DESCRIPTION.rst [hunk 4]
```diff
@@ -0,0 +1,5 @@
+[ This file must be max 2-3 paragraphs, and is required. ]
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/DESCRIPTION.rst]: Give a real description here.

File: l10n_nl_dutch_company_type/readme/HISTORY.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
+[ The change log. The goal of this file is to help readers
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/HISTORY.rst]: get rid of this file as well

File: l10n_nl_dutch_company_type/readme/ROADMAP.rst [hunk 6]
```diff
@@ -0,0 +1,5 @@
+[ Enumerate known caveats and future potential improvements.
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/ROADMAP.rst]: And get rid of this one

File: l10n_nl_dutch_company_type/readme/USAGE.rst [hunk 7]
```diff
@@ -0,0 +1,11 @@
+[ This file must be present and contains the usage instructions
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/USAGE.rst]: And this one.

**Review by NL66278 — CHANGES_REQUESTED** [review]
See comments

**Comment by lfreeke** [comment]
> yes, we can only go further here if all dependencies are merged

The dependency is merged. https://github.com/OCA/partner-contact/pull/660

**Comment by NL66278** [comment]
@hbrunn Can you remove the requisite branding label?

File: l10n_nl_dutch_company_type/__manifest__.py [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/__manifest__.py]: make it https

File: l10n_nl_dutch_company_type/readme/INSTALL.rst [hunk 9]
```diff
@@ -0,0 +1,8 @@
+[ This file must only be present if there are very specific
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/INSTALL.rst]: Please remove this file (and regenerate README)

File: l10n_nl_dutch_company_type/readme/DESCRIPTION.rst [hunk 10]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/DESCRIPTION.rst]: Put some real description here. Maybe a link to a wiki or to  standard where the dutch company types are described.

File: l10n_nl_dutch_company_type/readme/CONFIGURE.rst [hunk 11]
```diff
@@ -0,0 +1,13 @@
+[ This file is optional, it should explain how to configure
```
Comment (NL66278) [inline l10n_nl_dutch_company_type/readme/CONFIGURE.rst]: Get rid of the first three lines.

**Comment by lfreeke** [comment]
@NL66278 I resolved the comments.

File: l10n_nl_dutch_company_type/__manifest__.py (hunk 8 above)
Comment (NL66278) [reply l10n_nl_dutch_company_type/__manifest__.py]: But license is still http?

Comment (lfreeke) [reply l10n_nl_dutch_company_type/__manifest__.py]: Now it is https

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM + tested

File: l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot [hunk 12]
```diff
@@ -0,0 +1,130 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* l10n_nl_dutch_company_type
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
```
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot:7]: 11

File: l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot [hunk 13]
```diff
@@ -0,0 +1,130 @@
...
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: l10n_nl_dutch_company_type
+#: model:res.partner.company.type,name:l10n_nl_dutch_company_type.company_type_association
+msgid "Association"
+msgstr ""
```
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot:19]: no translation?

File: l10n_nl_dutch_company_type/i18n/nl.po [hunk 14]
```diff
@@ -0,0 +1,132 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* l10n_nl_dutch_company_type
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 10.0\n"
```
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/nl.po]: 11

File: l10n_nl_dutch_company_type/i18n/nl.po [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2018-02-01 12:56+0000\n"
+"PO-Revision-Date: 2018-02-01 12:56+0000\n"
+"Last-Translator: <>\n"
```
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/i18n/nl.po:11]: add name

**Review by gfcapalbo — CHANGES_REQUESTED** [review]

File: l10n_nl_dutch_company_type/README.rst [hunk 16]
```diff
@@ -0,0 +1,95 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to:
```
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/README.rst:38]: Delete or complete readme

File: l10n_nl_dutch_company_type/README.rst [hunk 17]
```diff
@@ -0,0 +1,95 @@
+==========================
+l10n_nl_dutch_company_type
+==========================
```
Comment (gfcapalbo) [inline l10n_nl_dutch_company_type/README.rst]: real name

File: l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot (hunk 13 above)
Comment (lfreeke) [reply l10n_nl_dutch_company_type/i18n/l10n_nl_dutch_company_type.pot:19]: No translation needed here, it is the pot file, translation are in nl.po

**Comment by lfreeke** [comment]
Is there a possibility that someone can rebuild this branch? Travis is giving this error 

psycopg2.OperationalError: could not connect to server: No such file or directory
Is the server running locally and accepting
connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?

**Comment by NL66278** [comment]
@lfreeke Update (in a separate commit) .travis.yml. Use 11.0 version of partner-contact as a guideline. At least update postgres to 9.6 instead of 9.2. Also add a MAKEPOT="0" to the line under global:

**Comment by NL66278** [comment]
@lfreeke Ping

**Comment by lfreeke** [comment]
@NL66278 see this merge request https://github.com/OCA/l10n-netherlands/pull/257 and this one https://github.com/OCA/l10n-netherlands/pull/258

**Pull request merged**